interface/patient_file/birthday_alert/birthday_pop.php

```diff
@@ -0,0 +1,53 @@
+<?php
+/**
+ *
+ * LICENSE: This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://opensource.org/licenses/gpl-license.php>;.
+ *
+ * @package OpenEMR
+ * @author  Sharon Cohen<sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
+
+
+/**picture free taken from https://pixabay.com/en/balloons-party-celebration-floating-154949*/
+include_once("../../globals.php");
+
+?>
+
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <script type="text/javascript" src="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-min-1-7-2/index.js"></script>
```

Comment:
remove above line and do the following instead:
``````
Header::setupHeader();
``````
This is a super cool function that @robertdown built to bring in assets. It automatically brings in jquery and bootstrap(and rtl bootstrap if pertinent), so you can now use jquery and bootstrap below.